src/mantarray_desktop_app/process_manager.py

```diff
@@ -36,7 +36,7 @@ def __init__(
     ) -> None:
         self._queue_container: MantarrayQueueContainer
 
-        self._ok_communication_process: OkCommunicationProcess
+        self._instrument_communication_process: OkCommunicationProcess
```

Comment:
Should this type definition be InstrumentProcess now? Or not ready to make that change yet

Reply:
I was planning to change that once we added the new comm process but I don't think there would be any problem doing that right now